mlir-tensorrt/README.md

```diff
@@ -35,46 +35,26 @@ bug fixes from [this open MLIR PR](https://github.com/llvm/llvm-project/pull/915
 
 Here we only show how to do Option 2.
 
-1. Build MLIR
-
-```sh
-# Clone llvm-project
-git clone https://github.com/llvm/llvm-project.git llvm-project
-
-# Checkout the right commit
-cd llvm-project
-git checkout $(cat ../build_tools/llvm_commit.txt)
-
-# Apply patch from llvm-project PR 91524
-git apply ../build_tools/llvm-project.patch
-
-# Do the build
-cd ..
-./build_tools/scripts/build_mlir.sh llvm-project build/llvm-project
-```
-
-2. Build the project and run all tests
-
 ```bash
-cmake -B ./build/mlir-tensorrt -S . -G Ninja \
+cmake -B build -S . -G Ninja \
```

Comment:
Build instructions here are within the development container, but we still do not have instructions on how to launch the development container. @shelkesagar29 are you planning to add them soon?

Reply:
yes it can be a separate PR